share/functions/nextd-or-forward-word.fish

```diff
@@ -1,9 +0,0 @@
-function nextd-or-forward-word
-    set -l cmd (commandline)
-    if test -z "$cmd"
-        nextd
-        commandline -f repaint
-    else
-        commandline -f forward-word
```

Comment:
thanks for the great work.

I had concern about losing introspectability, but the new docs in `man bind` make up for it (much easier to find than before).